tests/integration/queries/test_workflow.py

```diff
@@ -90,12 +90,77 @@ def test_workflow_submission(
     result = client.call(RetrieveStorageObject(result_url.result))
     assert isinstance(result, dict)
     assert result["submission_id"] == submission_id
+    assert result["file_version"] == 1
     client.call(UpdateSubmission(submission_id, retrieved=True))
     sub = client.call(GetSubmission(submission_id))
     assert isinstance(sub, Submission)
     assert sub.retrieved is True
 
 
+@pytest.mark.parametrize(
+    "_input",
+    [
+        {"urls": [PUBLIC_URL + "mock.pdf"] * 3},
+        {"files": [str(Path(__file__).parents[1]) + "/data/mock.pdf"] * 3},
+    ],
+)
+def test_workflow_submission_versioned(
+    indico, airlines_dataset, airlines_model_group: ModelGroup, _input
+):
+    client = IndicoClient()
+    wfs = client.call(ListWorkflows(dataset_ids=[airlines_dataset.id]))
+    wf = max(wfs, key=lambda w: w.id)
```

Comment:
kinda feels like we should put this into a fixture, also probably not trust whatever the latest workflow is on that dataset. Could be something crazy and we now have a test interdependence.